moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -674,75 +688,80 @@ double ServoCalcs::velocityScalingFactorForSingularity(const Eigen::VectorXd& co
 
 void ServoCalcs::enforceSRDFAccelVelLimits(Eigen::ArrayXd& delta_theta)
```

Comment:
let's definitely make sure the velocity limit enforcement gets updated, like we discussed. Here's the ROS1 PR:

ros-planning/moveit#2260